chord_metadata_service/patients/api_views.py

```diff
@@ -213,3 +213,72 @@ def get(self, request, *args, **kwargs):
                 "experiment_type": experiment_types
             }
         })
+
+
+class BeaconListIndividuals(APIView):
+    """
+    View to return lists of individuals filtered using search terms from katsu's config.json.
+    Uncensored equivalent of PublicListIndividuals.
+    """
+    def filter_queryset(self, queryset):
+        # Check query parameters validity
+        qp = self.request.query_params
+        if len(qp) > settings.CONFIG_PUBLIC["rules"]["max_query_parameters"]:
+            raise ValidationError(f"Wrong number of fields: {len(qp)}")
+
+        search_conf = settings.CONFIG_PUBLIC["search"]
+        field_conf = settings.CONFIG_PUBLIC["fields"]
+        queryable_fields = {
+            f"{f}": field_conf[f] for section in search_conf for f in section["fields"]
```

Comment:
i think this is a needless format string - it can just be `f:`

Reply:
Oh noes! You want me to pay attention to the code I'm copypasting? :sob:   

(fixed)